app/uk/gov/hmrc/economiccrimelevyreturns/navigation/UkRevenuePageNavigator.scala

```diff
@@ -32,7 +33,19 @@ class UkRevenuePageNavigator @Inject() (eclLiabilityService: EclLiabilityService
 
   override protected def navigateInNormalMode(eclReturn: EclReturn)(implicit request: RequestHeader): Future[Call] =
     eclReturn.relevantApRevenue match {
-      case Some(_) => Future.successful(routes.AmlRegulatedActivityController.onPageLoad(NormalMode))
+      case Some(_) =>
+        eclLiabilityService.calculateLiability(eclReturn) match {
+          case Some(f) =>
+            f.map { updatedReturn =>
+              updatedReturn.calculatedLiability match {
+                case Some(calculatedLiability) if calculatedLiability.calculatedBand == Small =>
+                  routes.AmountDueController.onPageLoad(NormalMode)
+                case Some(_)                                                                  => routes.AmlRegulatedActivityController.onPageLoad(NormalMode)
+                case _                                                                        => routes.NotableErrorController.answersAreInvalid()
+              }
+            }
+          case None    => Future.successful(routes.NotableErrorController.answersAreInvalid())
+        }
```

Comment:
Do we need similar logic in the check mode? What if I change my answer and am no longer falling into the small band?

Reply:
You mean that if I change my answer such that I now fall into a liable band size (Medium, Large or VeryLarge) from Small then we'll need to direct to the AML activity for full FY question as we will not have captured that?

That raises another point I think; if I do the reverse (i.e. change my answer such that I now fall into the Small band size) then we'll need to clear out the AML activity related answers.